changelogs/changelog.yaml

```
@@ -80,3 +80,22 @@ releases:
+          quietly without a config file.
+        - Treat `host` as a string (not a path) to prevent URLs from being coerced into
+          file paths, fixing "No host specified" errors.
+      minor_changes:
+        - Added support for Basic auth (username/password) in the inventory plugin for
+          environments with proxies that accept HTTP Basic.
+        - Improved documentation and examples for `flightctl_config_file` (FlightCtl
+          config) and config precedence.
+      release_summary: Improved the FlightCtl inventory plugin robustness and configuration clarity.
+    fragments:
+      - flightctl_1.2.1_update.yml
+    release_date: '2025-09-08'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)